src/runtime/server.ts

```diff
@@ -269,6 +278,38 @@ function initWebsocket(wss, withIDE:boolean) {
   });
 }
 
+function loadDatabases(description) {
+  var databases = {};
+  var dbs = description;
+  if (!Array.isArray(dbs)) dbs = [dbs];
+  var arrayLength = dbs.length;
+  let multi = new MultiIndex();
+  let changes =  new Changes(multi);
+
+  for (var i = 0; i < arrayLength; i++) {
+    var database = "init";
+    var prefix = undefined;
+    var terms = dbs[i].split(":")
+    var count = 0;
+
+    if (terms.length == 3) prefix = terms[count++];
+    if (terms.length == 2) database = terms[count++];
+    var data = fs.readFileSync(terms[count]).toString();
```

Comment:
Oh, and we need to wrap this in `if(fs.existsSync(<path>) { ... } else { /* handle error */ }` or we'll explode if the file is missing.